vue-instantsearch/server-side-rendering/src/main.js

```diff
@@ -4,6 +4,16 @@ import { createRouter } from './router';
 import { createServerRootMixin } from 'vue-instantsearch';
 import algoliasearch from 'algoliasearch/lite';
 import qs from 'qs';
+import _renderToString from 'vue-server-renderer/basic';
+
+function renderToString(app) {
+  return new Promise((resolve, reject) => {
+    _renderToString(app, (err, res) => {
+      if (err) reject(err);
+      resolve(res);
+    });
+  });
+}
```

Comment:
Looks like it wasn't just about bumping the dependency, but also updating the code. Why is this new code needed?

Reply:
Indeed ! In the newer versions `findResultsState` is required and it throws `Error: findResultsState requires `renderToString: (component) => Promise<string>` in the first argument.` when not provided.

So we need this extra bit of code to wrap it into a promise.